tests/Integration/Schema/Directives/BuilderDirectiveTest.php

```diff
@@ -79,12 +147,13 @@ public function testCallsCustomBuilderMethodOnFieldWithoutValue(): void
     }
 
     /**
-     * @param  \Illuminate\Database\Query\Builder|\Illuminate\Database\Eloquent\Builder  $builder
+     * @param  \Illuminate\Database\Query\Builder|Builder  $builder
+     * @param  int  $value
```

Comment:
Why not a real type?

Reply:
I wasn't quite sure `?int $value` is supported on all the php versions you support, added it now thanks.